<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>112</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>平滑度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ts_lb">
          <property name="text">
           <string>0.3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="tsSlider">
        <property name="value">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>控制点</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="grainBox">
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="cspline_btn">
      <property name="text">
       <string>生成曲线</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="clean_btn">
      <property name="text">
       <string>清除</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>帧数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="frame_spinBox">
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QComboBox" name="inter_Box">
      <item>
       <property name="text">
        <string>线性插值</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>矢量线性插值</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="v_Box">
      <item>
       <property name="text">
        <string>匀速</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>加速</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>减速</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="move_btn">
      <property name="text">
       <string>开始运动</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>图1：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_pts_1">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>图2：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_pts_2">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>224</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>提示：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="notice_lb">
      <property name="text">
       <string>请绘制第一幅图</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="imgButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>载入图片</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2.png</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
